docs/Admin.md

```diff
@@ -441,6 +442,24 @@ See The [Authentication documentation](./Authentication.md#customizing-the-login
 
 **Tip**: Before considering writing your own login page component, please take a look at how to change the default [background image](./Theming.md#using-a-custom-login-page) or the [MUI theme](#theme). See the [Authentication documentation](./Authentication.md#customizing-the-login-component) for more details.
 
+## `loginCallbackPage`
+
+If you want to customize the LoginCallback page, pass a component of your own as the `loginCallbackPage` prop. React-admin will display this component whenever the `/login-callback` route is called.
+
+```jsx
+import MyLoginCallbackPage from './MyLoginCallbackPage';
+
+const App = () => (
+    <Admin loginCallbackPage={MyLoginCallbackPage}>
+        ...
+    </Admin>
+);
+```
+
+You can also disable it completely along with the `/login-callback` route by passing `false` to this prop.
```

Comment:
Shouldn't we say it is the default value?

```suggestion
You can also disable it completely along with the `/login-callback` route by passing `false` to this prop. This corresponds to the default setting.
```

Reply:
I don't get this addition

Reply:
It is not the default value